### PR TITLE
Add AWS support to velero

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: modules/kubernetes/velero/main.tf [hunk 1]
```diff
@@ -20,53 +20,37 @@ terraform {
...
+  repository       = "https://vmware-tanzu.github.io/helm-charts"
+  chart            = "velero"
+  name             = "velero"
+  namespace        = "velero"
+  create_namespace = true
+  version          = "2.14.0"
+  values           = [local.values]
 }
 
 resource "helm_release" "velero_extras" {
```
Comment (simongottschlag) [inline modules/kubernetes/velero/main.tf:40]: Should we really run this if we don't need it?

File: modules/kubernetes/velero/variables.tf [hunk 2]
```diff
@@ -4,38 +4,37 @@ variable "cloud_provider" {
...
+    client_id                 = string,
+    resource_id               = string,
+    storage_account_name      = string,
+    storage_account_container = string
+  })
+  default = {
+    subscription_id           = "",
+    tenant_id                 = "",
+    resource_group            = "",
+    client_id                 = "",
```
Comment (simongottschlag) [inline modules/kubernetes/velero/variables.tf:21]: Why commas on only a few of them?

File: modules/kubernetes/velero/main.tf (hunk 1 above)
Comment (phillebaba) [reply modules/kubernetes/velero/main.tf:40]: we still need it as it adds the schedule

File: modules/kubernetes/velero/variables.tf (hunk 2 above)
Comment (phillebaba) [reply modules/kubernetes/velero/variables.tf:21]: I will remove them

**Review by simongottschlag — APPROVED** [review]
LGTM

**Pull request merged**